ios/RNGestureHandlerManager.mm

```diff
@@ -196,24 +210,82 @@ - (void)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
 
 #pragma mark Events
 
-- (void)sendTouchEvent:(RNGestureHandlerEvent *)event
+- (void)sendStateChangeEvent:(RNGestureHandlerStateChange *)event withActionType:(nonnull NSNumber *)actionType
 {
-    [_eventDispatcher sendEvent:event];
+    switch ([actionType integerValue]) {
```

Comment:
should just be `actionType` here and there is no need to use integerValue (I think)

Reply:
Done in 3c381aa2075734797c5a997d6708843494d93dce